refactor.js

```diff
@@ -0,0 +1,65 @@
+class BookLists {
+  constructor() {
+    this.bkList = JSON.parse(localStorage.getItem('library')) || [];
+  }
+
+  displayBooks() {
+    const bookTable = document.getElementById('tb-body');
+    let table = '';
+    this.bkList.forEach((book, i) => {
+      table += `
+    <tr>
+      <th scope="row">${i + 1}</th>  
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td><span id="${book.id}" onClick="remove(${book.id
+})" class="btn btn-danger btn-sm delete">X</span></td>
+    </tr>`;
+    });
+    bookTable.innerHTML = table;
+  }
+
+  // add a book to library array
+  addBook(title, author) {
+    const book = {
+      id: this.bkList.length + 1,
+      title,
+      author,
+    };
+
+    this.bkList = [...this.bkList, book];
+    localStorage.setItem('library', JSON.stringify(this.bkList));
+  }
+
+  // delete a book
+  removeBook(index) {
+    this.bkList = this.bkList.filter((b) => b.id !== index);
```

Comment:
[Optional] For naming convention best practice, I suggest you give a more descriptive name to your variables and parameters.

Reply:
@bimbolabuari  thanks we will fix this issue.